testConnectivity.py

```diff
@@ -416,6 +433,15 @@ def testConnectivity(args):
         pass
         printGreen("VFAT Communication Successfully Established")
 
+    if args.writePhases2File and args.firstStep <= 4:
+        fName = elogPath+'/Phases.log'
+        fPhases = open(fName,"w") 
+        for ohN in range(nOHs):
+            # Skip masked OH's
+            if( not ((args.ohMask >> ohN) & 0x1)):
+                continue
+            fPhases.write("[{0}]: {1}".format(ohN,dict_phases2Save[ohN]))
```

Comment:
will this *always* write 24 numbers per OH?  e.g. what happens if it can't find a valid phase, I don't remember if I filled some dummy value to `dict_phases2Save[ohN]` then.

Reply:
Yes this currently writes 24 numbers per OH. e.g.:
[4]: [6, 2, 2, 4, 4, 2, 6, 2, 2, 2, 3, 3, 4, 2, 2, 4, 2, 2, 7, 5, 2, 2, 2, 2]
.. is there a way for me to test with no valid phase?

Reply:
simplest would be to insert a bad value into `dict_phases2Save[ohN]`.  Phase = 15 will always be bad.

Reply:
ok checked with @bdorney -- the code wont actually execute if the phases were bad. Anyway updated this part to be ROOT friendly.

Reply:
Yeah no longer an concern.